docs/current_docs/manuals/developer/cookbook/snippets/secret-file/python/main.py

```diff
@@ -0,0 +1,23 @@
+from typing import Annotated
+
+import dagger
+from dagger import Doc, dag, function, object_type
+
+
+@object_type
+class MyModule:
+    @function
+    async def github_api(
```

Comment:
```suggestion
    async def github_auth(
```
to match function call via cli